docs/user-manual/modules/ROOT/pages/lifecycle.adoc

```diff
@@ -102,3 +102,7 @@ stop/start.
 If a route consumer does not support suspension, it will fallback and
 stop the route instead.
 
+[[Lifecycle-RelatedInfo]]
+== Related Info
+
+xref:controlbus-component.adoc[Take a look at the Control Bus Component]
```

Comment:
We're using Antora to generate the website from this content, syntax for xref links is explained [here](https://docs.antora.org/antora/2.0/asciidoc/page-to-page-xref/).

Since this link is pointing between two components (from user manual to component reference) the syntax is: 

```suggestion
xref:components::controlbus-component.adoc[Take a look at the Control Bus Component]
```